Make.versions

```diff
@@ -43,5 +43,5 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=12.15.0.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=5.15.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=13.4.0.$(IOS_COMMIT_DISTANCE)
+MAC_PACKAGE_VERSION=6.4.0.$(MAC_COMMIT_DISTANCE)
```

Comment:
`master` should be `x.3.0` (odd) until we branch `d16-4` so it's easy to distinguish the branch from the number

Considering that `d16-4` release date is likely after one (or maybe two) Xcode 11.x release we might even want to keep more _digits_ available. I suggest

* `d16-3` -> x.2.0
* `xcode11.1` -> x.4.0 (should be branched from `d16-3` after the `xcode11` merge)
* `xcode11.2` -> x.6.0
* `d16-4` -> x.8.0
* `master` (post 16-4 branch) -> x.9.0

c.c. @chamons